cli/CHANGELOG.md

```diff
@@ -668,3 +668,9 @@ _Released 1/24/2023_
 - Video output link in `cypress run` mode has been added to it's own line to
   make the video output link more easily clickable in the terminal. Addresses
   [#23913](https://github.com/cypress-io/cypress/issues/23913).
+
+**Features:**
+
+- Passing `--browser` flag alone will launch after test selection
+in the GUI, the browser will automatically launch. Addresses 
+[#22003](https://github.com/cypress-io/cypress/issues/22003)
```

Comment:
You asked in https://discord.com/channels/755913899261296641/1185979505404289115 why the Semantic Pull Request check is failing

You have added this "Features" section to release `12.4.0` at the bottom of this file.
It would need to be moved to the upcoming new version at the top of this file.

Reply:
🤦‍♂️😂 I didn't even realize there were Semantic version numbers in the file at all! Thank you for pointing that out! When I get back home in a little while, I will make that change. 

Do you have, by chance, any advice on running the tests? And which file I should add my test(s) to?

Thank you for your response!